edb/server/args.py

```diff
@@ -313,6 +314,10 @@ def _writer(status: str) -> None:
         '--auto-shutdown', type=bool, default=False, is_flag=True,
         help='shutdown the server after the last ' +
              'connection is closed'),
+    click.option(
+        '--auto-shutdown-after', type=float, default=0.0,
+        help='shutdown the server after N seconds of inactivity. '
```

Comment:
```suggestion
        help='shutdown the server after N seconds of inactivity '
             'after the last connection is closed`
```

`--auto-shutdown` essentially becomes `--auto-shutdown-after=0`.  Perhaps we should also deprecate it.  @1st1?

Reply:
changed. I deprecated and hid the `--auto-shutdown` option.